drltrace/drltrace_frontend.cpp

```diff
@@ -97,6 +97,16 @@ static droption_t<bool> op_print_ret_addr
 (DROPTION_SCOPE_CLIENT, "print_ret_addr", false, "Print library call's return address",
  "Print return addresses of library calls.");
 
+static droption_t<uint> op_unknown_args
+(DROPTION_SCOPE_CLIENT, "num_unknown_args", 2, "Number of unknown libcall args to print",
+ "Number of arguments to print for unknown library calls.  Specify 0 to disable "
+ "unknown args printing.");
+
+static droption_t<uint> op_max_args
+(DROPTION_SCOPE_CLIENT, "num_max_args", 6, "Maximum number of arguments to print",
```

Comment:
It's not ideal to duplicated these default values.  The plan is to switch drltrace.c to use droption also and share these definitions directly, right?